proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -748,6 +753,20 @@ message AjRajTablesRequest {
   repeated string columns_to_add = 6;
 }
 
+message MultiJoinInput {
+  TableReference source_id = 1;
+  repeated string columns_to_match = 3;
+  repeated string columns_to_add = 4;
+}
+
+message MultiJoinTablesRequest {
```

Comment:
Is now the time to plumb this through `io.deephaven.qst` / `io.deephaven.api.TableOperations`? If not, please create a ticket.

Reply:
Created https://github.com/deephaven/deephaven-core/issues/5158